providers/openstack/README.md

```diff
@@ -140,45 +170,61 @@ openstackclusterstackreleasetemplate.infrastructure.clusterstack.x-k8s.io/cspote
 
 ## Create the workload cluster resource (SCS-User/customer)
 
-Create and apply `cluster.yaml` file to the management cluster:
+To create a workload cluster you must configure the following things:
 
 ```bash
-cat <<EOF | kubectl apply -f -
+export CS_CLUSTER_NAME=cs-cluster
+export CS_POD_CIDR=192.168.0.0/16
+export CS_SERVICE_CIDR=10.96.0.0/12
+export CS_EXTERNAL_ID=ebfe5546-f09f-4f42-ab54-094e457d42ec # gx-scs
+export CS_CLASS_NAME=openstack-"${CS_NAME}"-"${CS_K8S_VERSION/./-}"-"${CS_VERSION}"
+export CS_K8S_PATCH_VERSION=3
+```
+
+Create and apply `cluster.yaml` file to the management cluster.
+
+Depending on your cluster-class and cluster addons, some more variables may have to be provided in the `spec.topology.variables` list.
+An error message after applying the `Cluster` resource will tell you if more variables are necessary.
+
+```bash
+cat > cluster.yaml <<EOF
 apiVersion: cluster.x-k8s.io/v1beta1
 kind: Cluster
 metadata:
-  name: cs-cluster
-  namespace: my-tenant
+  name: ${CS_CLUSTER_NAME}
+  namespace: ${CS_NAMESPACE}
   labels:
     managed-secret: cloud-config
 spec:
   clusterNetwork:
     pods:
       cidrBlocks:
-        - 192.168.0.0/16
+        - ${CS_POD_CIDR}
```

Comment:
cidrBlocks should not be a environment variable as this limits it to only one entry. They however should be an array of strings

Reply:
What do you propose should be done?

In my opinion this is only an example and to highlight that this a configuration I added the environment variable.

Should I leave it untouched?

Reply:
i don't really know, maybe we just keep it as a environment variable and add a comment to it. whoever end up using this quickstart guide probably knows how to add multiple entries

Reply:
Added comment to the environment variable example section